pontoon/base/migrations/0003_add_terminology_project.py

```diff
@@ -0,0 +1,131 @@
+# Generated by Django 2.2.11 on 2020-05-07 14:25
+
+from django.db import migrations
+from django.db.models import Count
+
+
+def create_terminology_project(apps, schema_editor):
+    # Collect localizable term data to create entities and translations from
+    Term = apps.get_model("terminology", "Term")
+    TermTranslation = apps.get_model("terminology", "TermTranslation")
+    terms = Term.objects.filter(do_not_translate=False, forbidden=False).exclude(
+        definition=""
+    )
+    term_translations = TermTranslation.objects.filter(term__in=terms)
+
+    Locale = apps.get_model("base", "Locale")
+    locales = Locale.objects.exclude(code__in=["en-US", "en"]).annotate(
+        term_count=Count("terms")
+    )
+
+    # Create the "Terminology" project
+    Project = apps.get_model("base", "Project")
+    project, _ = Project.objects.get_or_create(
```

Comment:
I wonder if this should fail if there's already an existing project with the `terminology` slug.

Also what happens if a different slug had the "Terminology" name?

Reply:
Good point, this should fail in case some instance uses the "Terminology" name or "terminology" slug already. That way they will have to (and will be able to) modify their existing projects before upgrading.